packages/create-sitecore-jss/src/templates/angular/server.bundle.ts

```diff
@@ -52,13 +52,17 @@ function renderView(
 
     if (parsedData) {
       state.sitecore = parsedData.sitecore;
+      state.language = parsedData.sitecore.context.language;
     }
 
     // parse the URL that's being handled by Sitecore so we can pass in the initial state to the app
     const routeParser = new JssRouteBuilderService();
     const jssRoute = routeParser.parseRouteUrl(path.split('/').filter((segment) => segment));
     state.serverRoute = jssRoute.serverRoute;
-    state.language = parsedViewBag.language || jssRoute.language;
```

Comment:
In the case of Experience Editor / Horizon, the viewbag would actually have the language.
https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/testData/ee-data.ts#L12